src/MegaPoker.sol

```diff
@@ -69,7 +69,7 @@ contract MegaPoker is PokingAddresses {
         (ok,) = univ2daieth.call(abi.encodeWithSelector(0x18178358));
         (ok,) = univ2wbtceth.call(abi.encodeWithSelector(0x18178358));
         (ok,) = univ2usdceth.call(abi.encodeWithSelector(0x18178358));
-        (ok,) = univ2daiusdc.call(abi.encodeWithSelector(0x18178358));
+        // (ok,) = univ2daiusdc.call(abi.encodeWithSelector(0x18178358));
```

Comment:
We should continue to poke this so that fees are accounted for so users can generate more Dai against their deposits.

Reply:
Done